Sources/Preferences/PreferencesWindowController.swift

```diff
@@ -18,29 +18,24 @@ public final class PreferencesWindowController: NSWindowController {
 		super.init(window: window)
 
 		window.title = String(System.localizedString(forKey: "Preferences…").dropLast())
+		window.titleVisibility = style.windowTitleVisibility
 		window.contentView = tabViewController.view
-
-		tabViewController.tabViewItems = viewControllers.map { viewController in
-			let item = NSTabViewItem(identifier: viewController.toolbarItemTitle)
-			item.label = viewController.toolbarItemTitle
-			item.image = viewController.toolbarItemIcon
-			item.viewController = viewController as? NSViewController
-			return item
-		}
-		tabViewController.tabStyle = .toolbar
-		tabViewController.transitionOptions = [.crossfade, .slideDown]
+		tabViewController.isCrossfadingTransitions = crossfadeTransitions
+		tabViewController.configure(preferences: preferences, style: style)
 	}
 
+	@available(*, unavailable)
 	public required init?(coder: NSCoder) {
-		super.init(coder: coder)
+		fatalError("init(coder:) has not been implemented")
 	}
 
-	public func showWindow() {
+	public func showWindow(preferenceIdentifier: PreferenceIdentifier? = nil) {
```

Comment:
What do you think about this? (Or another short word) Would make it more fluent and Swifty.

```suggestion
	public func showWindow(at preferenceIdentifier: PreferenceIdentifier? = nil) {
```

Reply:
Would you prefer `showPreference(preferenceIdentifier:)` to skip the parameter name so one can call `showPreference(.general)`? Or rather `show(preference: .general)`?

Reply:
I think I like `show(preference:)` most, in fact, now that names and identifiers have changed to be truly unique.

Reply:
> I think I like show(preference:) most, in fact, now that names and identifiers have changed to be truly unique.

👍 

But are we sure `preference` is clear enough? "Preference" could really mean **one setting*, as in **preferences** are many settings (user preferences), and **a preference** is one setting. So maybe we should go with "preferencePane" for everything? Or maybe you have a better naming?

Reply:
That reminds me of the initial naming struggle :)

Looked it up this time. Leaning on to the [Apple HIG](https://developer.apple.com/design/human-interface-guidelines/macos/app-architecture/preferences/), I think

- `Preference` is 1 setting (e.g. corresponds to a single UserDefaults key)
- `Preferences` are for the app's whole collection of settings, as per the HIG:
    > A **preferences** window is a modeless dialog [...]
- `PreferencePane` for the view controller pages
    >  Update the window's title to reflect the currently visible **preference pane.**

So `PreferencePane` works for me.